nvtabular/ops/categorify.py

```diff
@@ -1299,7 +1611,7 @@ def _encode(
     path,
     df,
     cat_cache,
-    na_sentinel=-1,
+    na_sentinel=0,
```

Comment:
How does this interact with using the 0th position to encode unknown values? I can imagine this either simplifying things or causing potential issues, but I'm not sure which is happening here.

Reply:
Huh, yeah, I'm confused too! I don't really understand why that would be configurable either, so it's tempting to remove it. I feel like this could be a Chesterton's Fence situation though. 😅